uat/codebuild/uat_linux_buildspec.yaml

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+version: 0.2
+phases:
+  install:
+    runtime-versions:
+      java: corretto11
+  build:
+    commands:
+      - mvn -U -ntp clean verify -f uat/pom.xml
+      - mvn -U -ntp verify -DskipTests=true
```

Comment:
I think we can remove this step, we don't need to build the base project to run the uats

Reply:
removed